code/modules/projectiles/projectile/bullets/revolver.dm

```diff
@@ -30,6 +30,43 @@
 	stamina = 30
 	sharpness = SHARP_NONE
 
+/obj/item/projectile/bullet/c38/ap
+	name = ".38 armor-piercing bullet"
+	damage = 18
+	armour_penetration = 15 //Not actually all that great against armor, but not *terrible*
+
+/obj/item/projectile/bullet/c38/frost //Basically Iceblax 2
+	name = ".38 frost bullet"
+	armour_penetration = -45 //x1.81 vs x1.43 for how much more effective armor is
+	var/temperature = 100
+
+/obj/item/projectile/bullet/c38/frost/on_hit(atom/target, blocked = 0)
+	if(blocked != 100)
+		if(isliving(target))
+			var/mob/living/L = target
+			L.adjust_bodytemperature(((100-blocked)/100)*(temperature - L.bodytemperature))
+	return ..()
+
+/obj/item/projectile/bullet/c38/talon
+	name = ".38 talon bullet"
+	damage = 12 //Tested on naked felinids, could never cause a wound type above open cut.
+	stamina = 18
+	sharpness = SHARP_EDGED
+
+/obj/item/projectile/bullet/c38/talon/on_hit(atom/target, blocked = 0)
+	if(blocked != 100)
+		if(ishuman(target) && damage > 7)
```

Comment:
please make the damage breakpoint it's own variable

Reply:
its*
(yeah sure give me a moment)